pkg/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,69 @@
+package bootstrap
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/weaveworks/pctl/pkg/runner"
+	profilesv1 "github.com/weaveworks/profiles/api/v1alpha1"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	// GitRepository is the git repository flux resource the installation uses
+	GitRepository profilesv1.GitRepository `json:"gitRepository,omitempty"`
+}
+
+var r runner.Runner = &runner.CLIRunner{}
+
+func CreateConfig(namespace, name, directory string) error {
+	out, err := r.Run("git", "-C", directory, "rev-parse", "--show-toplevel")
+	if err != nil {
+		if strings.Contains(string(out), "not a git repository") {
+			return fmt.Errorf("the target directory %q is not a git repository", directory)
+		}
+		return fmt.Errorf("failed to get git directory location: %w", err)
+	}
+
+	pctlDir := filepath.Join(strings.TrimSuffix(string(out), "\n"), ".pctl")
```

Comment:
If we don't include the folder ( why the folder? are we planning on storing something else in it other than the config? ), we could name this `.pctl.config.yaml`, then this whole upper part of the function is the same as below in the GetConfig and could be extracted into a common function.

Even so, there is some small amount of duplication in here, but I think that's fine.

Reply:
On second thought, I'm okay with the folder.

Reply:
>  why the folder? are we planning on storing something else in it other than the config

Potentially yes. Mainly just following the `.wego/` convention. Happy to change it if you feel strongly :smile: